webapp/public/db/en/db.json

```diff
@@ -765,9 +750,40 @@
           { "name": "page", "url": "https://arxiv.org/pdf/2408.14837"}
         ],
         "isFirstLevelComplete": false,
-        "publishDate": "Aug 27A, 2024",
+        "publishDate": "Aug 27, 2024",
         "tags": ["custom-engine", "machine-learning", "ml", "rendering"],
         "previewImgUrl": "https://raw.githubusercontent.com/zvolchak/can-it-run-doom/main/storage/images/gamengen-rendering.png"
+      },
+    {
+      "id": "1044",
+      "title": "Doom on a LED Billboard",
+      "description": "Doom on a truck mounted LED Billboard. This required lots of custom code to achieve, and a couple of custom Linux drivers, to be able to treat the sign as a large screen.",
+      "authors": [{
+        "name": "jgcertified",
+        "url": "https://www.youtube.com/@jgcertified"
+      }],
+      "sourcesUrl": [{ "name": "YouTube", "url": "https://www.youtube.com/watch?v=PxIGuMif1Nk"}],
+      "isFirstLevelComplete": false,
+      "publishDate": "July 28, 2014",
+      "tags": ["custom-drivers", "led-display"],
+      "previewImgUrl": "https://raw.githubusercontent.com/zvolchak/can-it-run-doom/main/storage/images/doom-on-a-truck.png"
+    },
+    {
+      "id": "1045",
+      "title": "Doom on a building",
+      "description": "Playing Doom E1M1 on the Schönherz Mátrix - an annual event of showcasing mostly animations on a building. Doom is running on GZDoom and streamed through HDMI, but we only used the Mátrix as a display. It has a size of 22.5 x 28.5 meters, with a resolution of 26 x 32 pixels.",
+      "authors": [{
+        "name": "Zoltán Bugir",
+        "url": "https://github.com/zoli90/"
+      }],
+      "sourcesUrl": [{ "name": "YouTube", "url": "https://www.youtube.com/live/eBSH0kA1RNQ&t=5287"}],
```

Comment:
On which minute is Doom in that video?

Reply:
In the meantime I got feedback from other people too that https://www.youtube.com/live/eBSH0kA1RNQ?si=mYQuPbXxfGQ_L6og&t=5287 does not jump to 1:28:08 automatically. I replaced it with https://www.youtube.com/watch?v=eBSH0kA1RNQ&t=5287